p2p/src/utils/mux.rs

```diff
@@ -106,7 +107,9 @@ impl<S: Sender, R: Receiver> Muxer<S, R> {
             // introduces a per-sender fairness slot that effectively increases
             // capacity when cloned per message.
             if let Some(sender) = self.routes.lock().unwrap().get_mut(&subchannel) {
-                let _ = sender.try_send((pk, bytes));
+                if let Err(e) = sender.try_send((pk, bytes)) {
```

Comment:
This is a subtle change from `p2p` where we block in this case (because we assume senders want reliable delivery of messages). Did this approach seem better or just a way to avoid the sender clone issue?